registry.json

```diff
@@ -48,6 +48,8 @@
   "backbone": "npm:backbone",
   "backbone.babysitter": "github:marionettejs/backbone.babysitter",
   "backbone.wreqr": "github:marionettejs/backbone.wreqr",
+  "big.js": "github:MikeMcl/big.js",
+  "bignumber.js": "github:MikeMcl/big.js",
```

Comment:
Oops, that's a typo, they are supposed to be two different repos. The repo
names have .js so I just left it. I'll be back tomorrow to fix it.

/#!/JoePea
On Jan 7, 2015 12:49 AM, "Guy Bedford" <notifications@github.com> wrote:

> In registry.json
> <https://github.com/jspm/registry/pull/100#discussion-diff-22575774>:
>
> > @@ -48,6 +48,8 @@
> >    "backbone": "npm:backbone",
> >    "backbone.babysitter": "github:marionettejs/backbone.babysitter",
> >    "backbone.wreqr": "github:marionettejs/backbone.wreqr",
> > +  "big.js": "github:MikeMcl/big.js",
> > +  "bignumber.js": "github:MikeMcl/big.js",
>
> It is best to only call something by one name in the registry. Can we go
> with one of big.js or bignumber.js. Also do we really need the .js in the
> name here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jspm/registry/pull/100/files#r22575774>.
>